### PR TITLE
Start does not equal install

### PR DESCRIPTION
**Pull request by ahmedalsudani**
@fjvallarino @tejohnso ready for review. Updating E2E tests associated with this change.

### CODE REVIEW

**Comment by fjvallarino** [comment]
:+1:

**Comment by tejohnso** [comment]
@ahmedalsudani What about the UI indicating "Install Rise Player"?  I thought we talked about changing that in unattended mode to further reduce confusion.

File: installer/ui/partials/installation-progress.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <h1 id="playerInstallationHeader" class="half-bottom">Install Rise Player</h1>
```
Comment (tejohnso) [inline installer/ui/partials/installation-progress.html]: @ahmedalsudani This shouldn't say "Install Rise Player" in unattended.

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**